validator/validate.py

```diff
@@ -62,13 +63,17 @@ def validate_deployment_files():
     Raises:
         Exception: If the configuration path has not been configured.
         Exception: If the configuration path does not exist
+
+    Returns:
+        list: A list with errors, if empty there were no errors validating the files
```

Comment:
Don't forget to add the corresponding type hint in the function's signature

Reply:
ahhh forgot about the typings! thanks 🤗